doc/doxygen/riot.doxyfile

```diff
@@ -818,30 +818,11 @@ EXCLUDE_SYMLINKS       = NO
 # exclude all test directories for example use the pattern */test/*
 
 EXCLUDE_PATTERNS       = */board/*/tools/* \
-                         */cpu/cortexm_common/include/core_cm*.h \
-                         */cpu/ezr32wg/include/ezr* \
-                         */cpu/*/include/atmel/* \
-                         */cpu/k60/include/MK60D10.h \
-                         */cpu/k60/include/MK60DZ10.h \
-                         */cpu/k64f/include/MK64F12.h \
-                         */cpu/kw2x/include/MKW22D5.h \
-                         */cpu/lpc11u34/include/LPC11Uxx.h \
-                         */cpu/lpc1768/include/LPC17xx.h \
-                         */cpu/lpc*/include/core_cm*.h \
+                         */vendor/* \
```

Comment:
nice cleanup here! :+1: 
Could you move it to the correct place to keep the list sorted?